wolfssl/wolfcrypt/settings.h

```diff
@@ -2671,6 +2671,11 @@ extern void uITRON4_free(void *p) ;
     #define HAVE_ONE_TIME_AUTH
 #endif
 
+/* Clients do not need a session cache. */
+#if defined(NO_WOLFSSL_SERVER)
+    #define NO_SESSION_CACHE_REF
```

Comment:
Please either add ` && !defined(NO_SESSION_CACHE_REF)` or do `#undef NO_SESSION_CACHE_REF` to avoid a redefinition warning.

Reply:
Good catch!! Fixed.